ci/build-project.ps1

```diff
@@ -0,0 +1,49 @@
+param(
+    [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine", ## This should ultimately resolve to "C:\b\<number>\UnrealEngine".
+    [string] $test_repo_branch,
+    [string] $test_repo_url,
+    [string] $test_repo_uproject_path,
+    [string] $test_repo_path,
+    [string] $msbuild_exe,
+    [string] $gdk_home,
+    [string] $build_platform,
+    [string] $build_state,
+    [string] $build_target
+)
+
+# Clone the testing project
+Write-Output "Downloading the testing project from $($test_repo_url)"
+git clone -b "$test_repo_branch" "$test_repo_url" "$test_repo_path" --depth 1
+if (-Not $?) {
+    Throw "Failed to clone testing project from $test_repo_url."
+}
+
+# The Plugin does not get recognised as an Engine plugin, because we are using a pre-built version of the engine
+# copying the plugin into the project's folder bypasses the issue
+New-Item -ItemType Junction -Name "UnrealGDK" -Path "$test_repo_path\Game\Plugins" -Target "$gdk_home"
+
+# Disable tutorials, otherwise the closing of the window will crash the editor due to some graphic context reason
+Add-Content -Path "$unreal_path\Engine\Config\BaseEditorSettings.ini" -Value "`r`n[/Script/IntroTutorials.TutorialStateSettings]`r`nTutorialsProgress=(Tutorial=/Engine/Tutorial/Basics/LevelEditorAttract.LevelEditorAttract_C,CurrentStage=0,bUserDismissed=True)`r`n"
+
+Write-Output "Generating project files"
+$proc = Start-Process "$unreal_path\Engine\Binaries\DotNET\UnrealBuildTool.exe" -Wait -ErrorAction Stop -NoNewWindow -PassThru -ArgumentList @(`
+    "-projectfiles", `
+    "-project=`"$test_repo_uproject_path`"", `
+    "-game", `
+    "-engine", `
+    "-progress"
+)
```

Comment:
Agreed, got rid of it! 💀

Reply:
I'm keeping the two `Start-Proces`s in `run-tests.ps1` though. Changing those to `&`s causes errors. I imagine it might have something to do with them opening a window.